UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common/Configuration/RadioNavigationalWarningsContextConfiguration.cs

```diff
@@ -0,0 +1,8 @@
+﻿
+namespace UKHO.MaritimeSafetyInformation.Common.Configuration
+{
```

Comment:
please exclude this class from code coverage. 
[ExcludeFromCodeCoverage]

Reply:
thanks for highlighting , done required changes.